vendor/github.com/grafana/loki/pkg/push/push-rf1.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
```

Comment:
What's that for ?

Reply:
Left over from the base PR. Not sure why git is picking up on it! But it was so we could have a separate grpc service for this new ingester, which is needed so traffic is routed correctly.